### ISSUE TITLE
--access checkout does ignore rollups

### ISSUE THREAD

**Issue by matthiaskrgr** [rust-lang/cargo-bisect-rustc#322]
when working with `--access checkout`, it seems like cbr still sends api calls to github in order to figure out the commits contained in a rollup instead of using the cloned repository.

**Comment by ehuss** [comment]
I don't think it is possible to get the information from the repository. It has to access the GitHub comment posted by the perf bot to know which artifacts to download.
